apps/services/auth-api/src/app/app.module.ts

```diff
@@ -11,8 +11,8 @@ import { UsersModule } from './modules/users/users.module'
   imports: [
     AuthModule.register({
       audience: '@identityserver.api',
-      issuer: process.env.IDS_ISSUER, // develop: https://identity-server.dev01.devland.is
-      jwksUri: process.env.JWKS_URI, //  develop: http://web-identity-server/.well-known/openid-configuration/jwks
+      issuer: process.env.IDS_ISSUER || "", // develop: https://identity-server.dev01.devland.is
+      jwksUri: process.env.JWKS_URI || "", //  develop: http://web-identity-server/.well-known/openid-configuration/jwks
```

Comment:
This should be coming from the environments file

Reply:
error TS2322: Type 'string | undefined' is not assignable to type 'string'.

I added the ' || "" ' part so that the tests will run and error TS2322 does not pop up.